code/world.dm

```diff
@@ -56,12 +56,12 @@
 	if(config.sql_enabled)
 		if(SSdbcore.Connect())
 			log_world("Database connection established.")
-			var/datum/DBQuery/query_feedback_create_round = SSdbcore.NewQuery("INSERT INTO [format_table_name("feedback")] SELECT null, Now(), IFNULL(MAX(round_id),0)+1, \"server_ip\", 0, \"[world.internet_address]:[world.port]\" FROM [format_table_name("feedback")]")
-			query_feedback_create_round.Execute()
-			var/datum/DBQuery/query_feedback_max_id = SSdbcore.NewQuery("SELECT MAX(round_id) FROM [format_table_name("feedback")]")
-			query_feedback_max_id.Execute()
-			if(query_feedback_max_id.NextRow())
-				GLOB.round_id = query_feedback_max_id.item[1]
+			var/datum/DBQuery/query_round_start = SSdbcore.NewQuery("INSERT INTO [format_table_name("round")] (start_datetime, server_ip, server_port) VALUES (Now(), INET_ATON('[world.internet_address]'), '[world.port]')")
+			query_round_start.Execute()
+			var/datum/DBQuery/query_round_max_id = SSdbcore.NewQuery("SELECT MAX(id) FROM [format_table_name("round")]")
+			query_round_max_id.Execute()
```

Comment:
This is a race condition that we can finally fix! If both servers start a new round at the same time, this could select the same id on both servers, leaving both servers to use the same round id. use  `SELECT LAST_INSERT_ID()`